manifest.json

```diff
@@ -6,6 +6,7 @@
     "default_icon": "icon.png"
   },
   "permissions": ["activeTab", "scripting"],
+  "host_permissions": ["<all_urls>"],
```

Comment:
This adds host_permissions to ensure the extension can access the audio context on any website.